pkgs/desktops/gnome-3/apps/file-roller/default.nix

```diff
@@ -24,6 +24,11 @@ stdenv.mkDerivation rec {
     patchShebangs data/set-mime-type-entry.py
   '';
 
+  postFixup = ''
+    wrapProgram "$out/bin/file-roller" \
+      --prefix PATH : ${lib.makeBinPath [ unzip ]}
```

Comment:
How is cpio invoked? Won’t it be needed here as well?

Reply:
fixed. thanks.